src/rviz/image/mouse_click.cpp

```diff
@@ -0,0 +1,127 @@
+#include "rviz/image/mouse_click.h"
+
+#include <ros/names.h>
+
+
+namespace rviz
+{
+
+MouseClick::MouseClick() : QObject()
```

Comment:
Provide a proper parent in the constructor as suggested in https://github.com/ros-visualization/rviz/pull/1737#discussion_r934052113:
```suggestion
MouseClick::MouseClick(QObject* parent) : QObject(parent)
```

Reply:
I have done this but it had an implication which I am not sure is ok.

Because I only get the parent **render_panel_**  during the ImageDisplay::onInitialize() I have to call the cnstructor of the MouseClick from there, instead of the ImageDisplay::ImageDisplay() ...